reactive/kubernetes_master.py

```diff
@@ -1377,11 +1377,16 @@ def send_data():
 @when('config.changed.extra_sans', 'certificates.available',
       'kube-api-endpoint.available')
```

Comment:
`@when` is a synonym of `@when_all` [1], so all states need to be set (re: a condition on line 1380) or the handler won't be run.

1. https://github.com/juju-solutions/charms.reactive/blob/5c8d9f35f484166b5277a1d922868af8fd8b39eb/charms/reactive/decorators.py#L113

Reply:
`@when_any` would make the handler run when any of the flags is set.